### DISCUSSION TITLE
Syntax error reported when using with Cypress

### DISCUSSION THREAD

**Discussion by Phonesis** [percy/cli#1048] [Support]
I am trying to use Percy with the CircleCI Cypress orb. 

When I run it I get the following error: 

```
> core-js-pure@3.24.1 postinstall /root/project/node_modules/core-js-pure
> node -e "try{require('./postinstall')}catch(e){}"


> @percy/core@1.10.0 postinstall /root/project/node_modules/@percy/core
> node ./post-install

(node:130) ExperimentalWarning: The ESM module loader is experimental.
file:///root/project/node_modules/@percy/core/post-install.js:6
    await import('./dist/install.js').then(install => install.chromium());
    ^^^^^

SyntaxError: Unexpected reserved word
    at Loader.moduleStrategy (internal/modules/esm/translators.js:117:18)
    at async link (internal/modules/esm/module_job.js:42:21)
```

Could this be an issue with the latest Percy CLI binary? I am following the percy usage instructions for Cypress orb here: 
https://circleci.com/developer/orbs/orb/cypress-io/cypress#usage-custom-command-prefix

**Comment by Robdel12** [comment]
👋🏼 hey @Phonesis! What version of node are you using? This looks like it’s a node error from an older version of Node.

**Comment by Phonesis** [comment]
@Robdel12 Tried both 12 and 14 via the Cypress orb executor commands. Not sure how else I can set it. 

I have tried removing percy/cli from my package.json and it gets further... percy launches when it is called here: 

```
      - cypress/run:
          name: cypress-tests
          executor: with-chrome
          browser: chrome
          command-prefix: npx percy exec --
          start: npm start
```

However. no snapshots are uploaded to the Percy server (PERCY_TOKEN is set in CI) 

Am I missing something?

**Reply by wwilsman** [reply]
@Phonesis we only support LTS versions of Node for as long as they are supported by Node itself. Since Node 12 reached EOL earlier this year, it is not supported.

By uninstalling `@percy/cli`, when you run `npx percy` it will actually download the `percy` package (the old deprecated package). The reason why this is getting "further" for you is because you are now using an outdated package with an outdated Node version. And the reason there are no snapshots uploaded is because newer SDK versions do not work with the outdated package anymore.

Using `@percy/cli` and the latest version of Node 14 (v14.20.0) should work fine since this is what all of our packages are developed against. If you're still getting that same `Unexpected reserved word` error, then Node may not be fully up-to-date.
